.npmignore

```diff
@@ -20,6 +20,4 @@ examples/
 instrumented/
 products/
 reports/
-tests/
-!tests/lib
-!tests/test-core
+tests/*-tests
```

Comment:
This is leaving in all-tests.html and testem.js, both of those should also be removed.